CHANGELOG.md

```diff
@@ -2,20 +2,27 @@
 
 Full documentation for Composable Kernel is not yet available.
 
-## (Unreleased) CK
+## CK for ROCm 6.1.0
```

Comment:
Looks good. Recommended order for consistency:

- Additions
- Optimizations
- Fixes

Reply:
Let's start with this release; for now, I have updated the ROCm 6.1 changelog manually in the correct order.

Reply:
OK, sounds good.